module-4/backend/week_one.md

```diff
@@ -5,15 +5,40 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+  - I liked event targeting and using that to access the desired attributes and use that in a variety of ways.
 2. What are some tools to help debug JavaScript code?
+  - Browser Dev Tools, debugger, pryjs, console.log(), etc.
 3. What are some tools you need in order to unit test your JavaScript?
+  - Mocha, Chai
 4. What is the syntax for invoking a function?
+```javascript
+function exampleFunction() {
+  // ... do a thing
+};
+
+exampleFunction(); // <--- invoking
+```
 5. What's `this` in JavaScript?
+  - Similar to `self` in Ruby. Kind of a snapshot of whatever is being interacted with, often (if not always) with the ability to invoke functions and attributes that belong to that object.
 6. What is Webpack and why is it useful?
+  - Webpack compiles your JS and CSS into one file for you, so you can abstract your code into seperate files then import them in one place.
 7. When/why do you want to use event delegation?
+  - To identify what is being interacted with/their parent element(s). You would use it in a situation where you need some information stored as an attribute to include in your code.
```

Comment:
also if you're trying to bind an event to an element that's not on the page on document ready!